### ISSUE TITLE
BUILD_ROOT + BUILD_INSTALLR_ROOT vars

### ISSUE THREAD

**Issue by maxnbk** [nerdvegas/rez#423]
This information is interpretable in other ways, but requires unneeded string manipulation.

REZ_BUILD_INSTALL_PATH points to the installing-variant's build path, e.g. /path/to/pkg/vers/variant
and REZ_BUILD_PATH does the same:
/path/to/builds/pkg/build/variant

however if you need the unvarianted root for some reason during your build, it requires some hoops as mentioned, because REZ_PKGNAME_ROOT is not available yet for the package being built.
Would be nice to have a:
REZ_BUILD_INSTALL_ROOT = /path/to/pkg/vers
REZ_BUILD_ROOT = /path/to/builds/pkg/build

**Comment by nerdvegas** [comment]
Agreed, though in keeping with existing rez terminology these vars should
be REZ_BUILD_INSTALL_BASE and REZ_BUILD_BASE.

Thx
A
